### DISCUSSION TITLE
別想逃稅！日、法 AI 查稅有成，固定資產稅疫後「超徵」有餘

### DISCUSSION THREAD

**Discussion by Lin-NZ** [1101-GS3073-C/Python-Community-GS3073-C#721] [科技新知]
content : 
https://www.inside.com.tw/article/30370-AI-tech-increase-Jp-and-Fr-tax
Reflections / Thoughts:
雖然目前準確率還有待提升，但帶來的稅務收益實在太高了，感覺政府部門會逐漸注重徵稅型AI的開發

**Comment by yjpxi** [comment]
在疫情期間，全球經濟受到衝擊，但房地產市場在許多國家仍然表現出色，在這種背景下，許多政府利用 AI 技術精確徵收固定資產稅，結果往往是超出預期的收入，出現所謂的「超徵」。
